src/OmniSharp.DotNetTest/VSTestManager.cs

```diff
@@ -148,15 +148,6 @@ public override async Task<GetTestStartInfoResponse> GetTestStartInfoAsync(strin
             VerifyTestFramework(testFrameworkName);
 
             var testCases = await DiscoverTestsAsync(new string[] { methodName }, runSettings, targetFrameworkVersion, cancellationToken);
-
-            SendMessage(MessageType.GetTestRunnerProcessStartInfoForRunSelected,
-                new
-                {
-                    TestCases = testCases,
-                    DebuggingEnabled = true,
-                    RunSettings = LoadRunSettingsOrDefault(runSettings, targetFrameworkVersion)
-                });
```

Comment:
This should have been removed when I extracted out the GetTestRunningProcessStartInfo method